pkgs/tools/system/syslog-ng/default.nix

```diff
@@ -100,7 +99,7 @@ stdenv.mkDerivation rec {
     "--enable-dynamic-linking"
     "--enable-systemd"
     "--enable-smtp"
-    "--with-python-packages=system"
```

Comment:
I'm curious about the difference this makes: it seems to have the same consequences in https://github.com/syslog-ng/syslog-ng/blob/449f16278a61ff83e069190b4850002d7c7c68d7/configure.ac#L1740-L1749

Reply:
At some point, I think `modules/python-modules/Makefile.am` tries to run setup.py and pip, unless `[ "$(with_python_packages)" != "none" ]`.

Reply:
did you check that it installs as much, as opposed to disabling building/installing python modules ?

Reply:
It does install the python modules in `lib/syslog-ng/python` as far as I can tell.